app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -105,11 +109,58 @@ class MainFragment : Fragment() {
                     .navigate(MainFragmentDirections.actionMainFragmentToNewProfileFragment())
             }
         }
-
+        currentlyActivePeofile()
         setHasOptionsMenu(true)
         return binding.root
     }
 
+    private fun currentlyActivePeofile() {
+        binding.activeProfile.visibility = readInt("ACTIVE_PROFILE_VISIBILITY")
+        binding.activeName.text = readString("ACTIVE_PROFILE_NAME")
+        binding.endTimeTxt.text = readString("END_TIME_TEXT")
+        binding.vibIconImg.setImageResource(readInt("VIBRATE_STATE_ICON"))
+
+        if (StartAlarm.beginStatus == 1) {
+            binding.activeProfile.visibility = View.VISIBLE
+            writeInt("ACTIVE_PROFILE_VISIBILITY", View.VISIBLE)
+
+            binding.activeName.text = "Active Profile: " + StartAlarm.activeProfile
+            writeString("ACTIVE_PROFILE_NAME", "Active Profile: " + StartAlarm.activeProfile)
```

Comment:
Please read the previous comment regarding the same carefully.
I asked you to define the string constants in a file of type object. You are supposed to reference these strings from that file then.